plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -247,6 +247,16 @@ class FossId internal constructor(
                     null
                 }
 
+                // This case could happen, when FossID project is created by user other than ORT user, and access rights
+                // are not granted to ORT user.
+                error == "Project does not exist or permission denied." && status == 0 -> {
+                    logger.error {
+                        "Project '$projectCode' does not exist or user '${config.user.value}' has no access to it."
+                    }
+
+                    error("Project '$projectCode' does not exist or user '${config.user.value}' has no access to it.")
```

Comment:
So, what exactly is the value of this change?

Before this change, the `IOException` below was thrown. After this change, an `error` with a different message is thrown. So basically this just seem to adjust the exception message to a more specialized one.

Can't the same be achieved by adding more information to the `IOException` below, e.g. by saying

    "Could not get project '$projectCode' for user '${config.user.value}: $error'"

That would avoid the fragile comparison against the exact error message, and also add that additional information for all other errors.

Reply:
One point though: this PR adds an extra logging output when user has not access to the project.
This is the important part of the change.

Reply:
Could this also be generalized so that the `else` case always logs all errors?

Reply:
Fine with me.

Reply:
Can I expect to get a fixup within a few hours to include it to a patch release for today?

Reply:
@kamil-bielecki-bosch, I believe the idea was to use exactly my sentence from [above](https://github.com/oss-review-toolkit/ort/pull/10335/files#r2090961940) for both the error log and as the exception message in the `else` branch, and in exchange drop the extra check for "Project does not exist or permission denied". Do we have a common understanding here, @nnobelis?